src/components/widgets/accounts-cash-projection/accounts-cash-projection.directive.coffee

```diff
@@ -28,6 +30,14 @@ module.controller('WidgetAccountsCashProjectionCtrl', ($scope, $q, $filter, Impa
     label: 'Get alerted when the cash projection goes below'
   }
 
+  # Define metadata for overdue transactions ranges
+  w.metadata.ranges = [
+    { name: '>60 days', to: moment().subtract(61,'d').format('YYYY-MM-DD') },
+    { name: '30-60 days', from: moment().subtract(60,'d').format('YYYY-MM-DD'), to: moment().subtract(30,'d').format('YYYY-MM-DD') },
+    { name: '<30 days', from: moment().subtract(29,'d').format('YYYY-MM-DD'), to: moment().format('YYYY-MM-DD') }
+  ]
```

Comment:
Do these params get applied when you save the widget settings? That was the reason I added the `toMetadata` & `initialize` method.

Reply:
Looks like they are being applied now =\